prospector/commit_processor/feature_extractor.py

```diff
@@ -7,11 +7,22 @@ def extract_features(commit: Commit, advisory_record: AdvisoryRecord) -> CommitF
     references_vuln_id = extract_references_vuln_id(
         commit.cve_refs, advisory_record.vulnerability_id
     )
+    changes_relevant_path = extract_changes_relevant_path(
+        advisory_record.paths, commit.changed_files
+    )
     commit_feature = CommitFeatures(
-        commit=commit, references_vuln_id=references_vuln_id
+        commit=commit,
+        references_vuln_id=references_vuln_id,
+        changes_relevant_path=changes_relevant_path,
     )
     return commit_feature
 
 
-def extract_references_vuln_id(cve_references: "list[str]", cve_id: str) -> str:
+def extract_references_vuln_id(cve_references: "list[str]", cve_id: str) -> bool:
     return cve_id in cve_references
+
+
+def extract_changes_relevant_path(
```

Comment:
Please provide a sentence to explain what this is supposed to do

Reply:
It is supposed to indicate that the commit changes at least one path that matches with the paths mentioned in the advisory record. Maybe we should rename this function along with the field of CommitFeature class to a more proper one. Let me know if you have any idea!